### PR TITLE
Add note about Yocto release schedules

### PR DESCRIPTION
**Pull request by drewmoseley**
Signed-off-by: Drew Moseley <drew.moseley@northern.tech>

### CODE REVIEW

**Comment by drewmoseley** [comment]
@lluiscampos somehow this one got missed in the recent docs pr.  Can you take a look?

File: 04.Artifacts/10.Yocto-project/01.Building/docs.md [hunk 1]
```diff
@@ -158,6 +158,8 @@ i.e. the top level of the Yocto Project build tree, and run these commands:
 ```bash
 git clone -b master git://github.com/mendersoftware/meta-mender
 ```
+<!--AUTOVERSION: "the HEAD of the %s branch"/meta-mender-->
```
Comment (kacf) [inline 04.Artifacts/10.Yocto-project/01.Building/docs.md]: It is just `%`, not `%s`.

Comment (lluiscampos) [reply 04.Artifacts/10.Yocto-project/01.Building/docs.md]: This was the reason for my re-submit of the original PR #917

**Comment by lluiscampos** [comment]
> @lluiscampos somehow this one got missed in the recent docs pr. Can you take a look?

The note is there: https://docs.mender.io/2.3/artifacts/yocto-project/building#adding-meta-mender-to-existing-yocto-project-environment Maybe you didn't refresh properly the site or something?

**Comment by drewmoseley** [comment]
Yeah, it was a git merge failure on my part.  I forgot you modified it.

**Pull request closed without merging**